pages/business/sales.js

```diff
@@ -25,11 +25,10 @@ const ProfileComponent = ({ }) => {
 
```

Comment:
formatting

Reply:
added plugin in vs code and run control+s in all files. Let me know if you want me change any settings in vs code configuration file?